### DISCUSSION TITLE
community/mirror/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4590] [Announcements]
# community/mirror/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/community/mirror/

**Comment by AkarinLiu** [comment]
求收录OpenResty镜像。
镜像名： openresty/openresty
简介：通过 Lua 扩展 NGINX 实现的可伸缩的 Web 平台

**Reply by windsonsea** [reply]
Hi @AkarinLiu，可以直接去 repo 里提 PR 哈，参见 [public-image-mirror readme](https://github.com/DaoCloud/public-image-mirror?tab=readme-ov-file#%E5%AE%9A%E6%9C%9F%E5%90%8C%E6%AD%A5%E5%88%97%E8%A1%A8)

**Comment by MagicSimba** [comment]
coredns 无论什么版本都报错，但是其他镜像却没有问题
time="2024-10-28T14:01:00Z" level=fatal msg="pulling image: rpc error: code = InvalidArgument desc = failed to pull and unpack image \"m.daocloud.io/registry.k8s.io/coredns:v1.10.1\": Target.MediaType must be set: invalid argument

**Reply by jzhupup** [reply]
正确的应该是：m.daocloud.io/registry.k8s.io/coredns/coredns:v1.10.1，少写了一层

**Reply by MagicSimba** [reply]
kubeadm 默认拉取的是 m.daocloud.io/registry.k8s.io/coredns:v1.10.1，我把配置文件 kubeadm.yaml 中 ClusterConfiguration.dns.imageRepository 改成 m.daocloud.io/registry.k8s.io/coredns 成功拉取到镜像了

**Comment by reckless-huang** [comment]
daocloud.io 无法使用了

**Comment by DouHuanHuan** [comment]
403 Error
```text
docker pull m.daocloud.io/docker.io/heroku/builder:24
Error response from daemon: unknown: failed to resolve reference "m.daocloud.io/docker.io/heroku/builder:24": unexpected status from HEAD request to https://m.daocloud.io/v2/docker.io/heroku/builder/manifests/24: 403 Forbidden
```

**Comment by thxcn** [comment]
docker pull container-registry.oracle.com/database/free:latest
docker pull mcr.microsoft.com/mssql/server:2025-latest
像是这两个应该怎么加速啊
